ui-components/src/helpers/filters.ts

```diff
@@ -31,6 +31,7 @@ function getComparisonForFilter(filterKey: ListingFilterKeys) {
       return EnumListingFilterParamsComparison["IN"]
     case ListingFilterKeys.seniorHousing:
     case ListingFilterKeys.availability:
+    case ListingFilterKeys.include_nulls:
```

Comment:
This won't be necessary once we remove `include_nulls` from `ListingFilterKeys`.